src/wix/WixToolset.Core/ExtensibilityServices/ParseHelper.cs

```diff
@@ -423,13 +423,27 @@ public Identifier GetAttributeIdentifier(SourceLineNumber sourceLineNumbers, XAt
                         access = AccessModifier.Section;
                         break;
 
+                    case "virtual":
+                        access = AccessModifier.Virtual;
+                        break;
+
+                    case "override":
+                        access = AccessModifier.Override;
+                        break;
+
                     default:
                         return null;
                 }
 
                 value = value.Substring(separator + 1).Trim();
             }
 
+            //////if (!allowVirtual && (access == AccessModifier.Virtual || access == AccessModifier.Override))
```

Comment:
clean up

Reply:
Fixed locally. I will push a rebase at the end of your review @barnson.